.github/workflows/release.yml

```diff
@@ -0,0 +1,142 @@
+          helm plugin install https://github.com/helm-unittest/helm-unittest.git
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Create Docker Builder Config File
+        run: sudo touch /etc/buildkitd.toml
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - uses: actions/checkout@v4
+      - name: Split version
+        id: split_version
+        uses: ./.github/actions/split_version
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+          release-branch: ${{ env.RELEASE_BRANCH }}
+          existing-version: ${{ env.EXISTING_VERSION }}
+      - name: Print release parameters
+        run: | 
+          echo "Releasing with the following parameters"
+          echo "Release version: ${{ env.RELEASE_VERSION }}"
+          echo "Development version: ${{ env.DEVELOPMENT_VERSION }}"
+          echo "Minor version: ${{ steps.split_version.outputs.minor-version }}"
+          echo "Patch Version: ${{ steps.split_version.outputs.patch-version }}"
+          echo "Is pre release version: ${{ steps.split_version.outputs.is-pre-release }}"
+          echo "Release branch: ${{ steps.split_version.outputs.release-branch }}"
+          echo "Existing version: ${{ steps.split_version.outputs.existing-version }}"
+      - name: Checkout release branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.split_version.outputs.release-branch }}
+      - name: Update Antora Docs
+        uses: ./.github/actions/update_antora
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+          minor-version: ${{ steps.split_version.outputs.minor-version }} 
+          patch-version: ${{ steps.split_version.outputs.patch-version }}
+          is-pre-release: ${{ steps.split_version.outputs.is-pre-release }}
+          release-branch: ${{ steps.split_version.outputs.release-branch }}
+          custom-facet-old-version: ${{ steps.split_version.outputs.existing-version }}
+          custom-facet-new-version: ${{ env.RELEASE_VERSION }}
+      - name: Create settings.xml
+        run: |
+          echo "<settings><servers><server><id>ossrh</id><username>${{ secrets.SONATYPE_CENTRAL_REPO_TOKEN_USER }}</username><password>${{ secrets.SONATYPE_CENTRAL_REPO_TOKEN_KEY }}</password></server><server><id>ghcr.io</id><username>${{ secrets.GHCR_IO_USERNAME }}</username><password>${{ secrets.GHCR_IO_TOKEN }}</password></server><server><id>pypi</id><username>${{ secrets.PYPI_USERNAME }}</username><password>${{ secrets.PYPI_TOKEN }}</password></server><server><id>dev-pypi</id><username>${{ secrets.TEST_PYPI_USERNAME }}</username><password>${{ secrets.TEST_PYPI_TOKEN }}</password></server> </servers></settings>" > $HOME/.m2/settings.xml
```

Comment:
Q: Should we be using a differ pypi server for the release? I believe test pypi is only for our snapshots.

Reply:
Its being uploaded to pypi not test pypi, I verified this.

Reply:
Safe to remove this then? `<id>dev-pypi</id><username>${{ secrets.TEST_PYPI_USERNAME }}</username><password>${{ secrets.TEST_PYPI_TOKEN }}</password>`?

Reply:
I am not sure if it is safe to remove!

Reply:
yes it should be

Reply:
Removed